Frameworks/CoreGraphics/CGPath.mm

```diff
@@ -200,47 +213,46 @@ CGPathRef CGPathCreateCopy(CGPathRef path) {
 }
 
 /**
- @Status Interoperable
+@Status Interoperable
+@Notes
 */
 CGMutablePathRef CGPathCreateMutableCopy(CGPathRef path) {
     if (path == NULL) {
         return NULL;
     }
 
-    auto ret = __CGPath::alloc(nil);
-    ret->_max = path->_max;
-    ret->_count = path->_count;
-    ret->_elements = (CGPathElementInternal*)IwRealloc(ret->_elements, ret->_max * sizeof(CGPathElementInternal));
-    memcpy(ret->_elements, path->_elements, path->_count * sizeof(CGPathElementInternal));
-    // All of the new elements need to call init
-    // because they were created with alloc + memcpy
-    for (unsigned i = 0; i < path->_count; i++) {
-        ret->_elements[i].init();
+    CGMutablePathRef mutableRet = CGPathCreateMutable();
+
+    path->__closePath();
```

Comment:
Synced up offline:
1. Revisit the path re-editing code validity.
2. If the path is not "conceptually" mutated, then this is okay, but we need to be mindful of the side effects.  

---
In reply to: [83480290](https://github.com/Microsoft/WinObjC/pull/1138#discussion_r83480290) [](ancestors = 83480290,83345783)